src/systems/diff_drive/DiffDrive.cc

```diff
@@ -210,43 +210,69 @@ void DiffDrive::Configure(const Entity &_entity,
   this->dataPtr->limiterAng = std::make_unique<ignition::math::SpeedLimiter>();
 
   // Parse speed limiter parameters.
-  if (_sdf->HasElement("min_velocity"))
```

Comment:
We should keep the old parameters, otherwise current users will be broken. How about his behaviour:

1. If the `min_velocity` is set, use it for both linear and angular
2. If `min_linear_velocity` is set, override the value above just for linear. The same for angular.

Reply:
I have made these changes.